### PR TITLE
🎓 🤖 Asn2 Project Download and Compression Task

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
1. Add the project files for asn2
2. Remove solution bit and reorganize for them
3. Add compression task for asn2

### Why
They need this. 

### How
Copied from private repo, did some structural changes to the project. Removed the equals tests. 

### Testing
Yes, but before I deleted the solutions. 

### Additional Notes
There seems to be a way to group compression tasks together, but it's far too complex to care about right now.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,15 +25,22 @@ test {
...
+task compressAssignment1(type: Zip) {
     archiveFileName = "Assignment1.zip"
     destinationDirectory = file("src/site/data/")
     from "src/site/data/Assignment1"
 }
 
+task compressAssignment2(type: Zip) {
+    archiveFileName = "Assignment3.zip"
+    destinationDirectory = file("src/site/data/")
+    from "src/site/data/Assignment3"
```
Comment (twentylemon) [inline build.gradle:37]: Presumably this is supposed to be assignment 2

Comment (jameshughes89) [reply build.gradle:37]: what. the. f?

**Review by twentylemon — APPROVED** [review]
It do be a lot of files, tho.

**Comment by jameshughes89** [comment]
> It do be a lot of files, tho.

it dube. +1,500

**Pull request merged**